### DISCUSSION TITLE
今天也是通过奇怪方式学到知识的一天

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#569] [General]
![353A74D1-C56D-4266-95BF-9B6EDD8346AA](https://user-images.githubusercontent.com/9006264/148485774-5ce4e3bf-887f-4383-82b6-818856affc1a.jpeg)

**Comment by Arondight** [comment]
现在语言基本上都支持unicode字符变量很久了， https://github.com/program-in-chinese

**Reply by Arondight** [reply]
```
    新特征组 = 提取图片特征(img)
    if 新特征组 is not None: 
        特征组 = 新特征组 - 原点特征组
```

差不多是这种感觉，几个月前我看了个 vtuber 项目才知道有人真是这么写程序的，全篇汉字
